### PR TITLE
[IMP] point_of_sale: stripe add details

### PR DESCRIPTION
**Pull request by Ysoroko**
This PR adds details like countries coverage, supported terminals and payment methods to Stripe documentation for point_of_sale

task-3865904

### CODE REVIEW

File: content/applications/sales/point_of_sale/payment_methods/terminals/stripe.rst [hunk 1]
```diff
@@ -6,7 +6,11 @@ Connecting a payment terminal allows you to offer a fluid payment flow to your c
 the work of your cashiers.
 
 .. seealso::
-   - :doc:`Use Stripe as payment provider. <../../../../finance/payment_providers/stripe>`
+   - :doc:`Using Stripe as payment provider documentation
+     <../../../../finance/payment_providers/stripe>`
+   - `List of countries supported on Stripe <https://stripe.com/global>`_
```
Comment (fja-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/stripe.rst:11]: Payment Terminals are not supported in every one of these countries: https://support.stripe.com/questions/global-availability-for-stripe-terminal

**Comment by jcs-odoo** [comment]
@robodoo r+

**Comment by jcs-odoo** [comment]
@robodoo r-

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**